lib/Server.js

```diff
@@ -246,6 +243,8 @@ class Server {
       // It is possible to use the `bypass` method without a `target`.
       // However, the proxy middleware has no use in this case, and will fail to instantiate.
       if (proxyConfig.target) {
+        const { createProxyMiddleware } = require('http-proxy-middleware');
```

Comment:
Move it here https://github.com/webpack/webpack-dev-server/blob/d8ffe3927b235e542eae23bab6e9dfdbea8e5bf0/lib/Server.js#L173 better we still will use it due https://github.com/webpack/webpack-dev-server/blob/d8ffe3927b235e542eae23bab6e9dfdbea8e5bf0/lib/Server.js#L421

Reply:
On it.